models/vrp.rb

```diff
@@ -571,5 +575,13 @@ def calculate_service_exclusion_costs(type = :time, force_recalc = false)
     def transactions
       vehicles.count * points.count
     end
+
+    def scheduling?
+      !self.schedule_range_indices.nil?
+    end
+
+    def periodic?
```

Comment:
should we call this `periodic_heuristic?` instead? because we can have periodic vehicle routing problem without this heuristic, only if a user specifies `visits_number` in vrp...